### PR TITLE
[#2110] Use new password rules

### PR DESCRIPTION
**Pull request by k88hudson**

### CODE REVIEW

**Comment by k88hudson** [comment]
Not sure why the linter is all of a sudden not liking arrow functions

**Comment by k88hudson** [comment]
@gvn R?

File: www_src/lib/validators.js [hunk 1]
```diff
@@ -1,3 +1,18 @@
+var owasp = require('owasp-password-strength-test');
+
+// this removes the repeating character required test
+// https://github.com/cadecairos/PassTest/blob/master/index.js
+owasp.tests.required.splice(2, 1);
+
+//https://github.com/mozilla/id.webmaker.org/blob/develop/web/server.js#L13
+owasp.config({
+  minLength: 8,
+  maxLength: 256,
```
Comment (gvn) [inline www_src/lib/validators.js]: Whoa awesome.

**Comment by gvn** [comment]
Everything seems to be working...created a new account and was successful! R+

**Comment by k88hudson** [comment]
thanks :dancer:

**Pull request merged**